com.unity.shadergraph/Editor/Data/Legacy/AbstractMaterialNode0.cs

```diff
@@ -15,6 +15,8 @@ class AbstractMaterialNode0
 
         public string m_KeywordGuidSerialized;
 
+        public string m_DropdownGuidSerialized;
```

Comment:
again, will dropdowns be ported to old master nodes? If no, not needed

Reply:
Also removed, cheers.